frontend/src/components/admin/assessment-creation/AssessmentQuestions.tsx

```diff
@@ -23,7 +28,14 @@ const AssessmentQuestions = (): React.ReactElement => {
         <Spacer />
         <Button
           leftIcon={<PlusOutlineIcon />}
-          onClick={() => setShowQuestionEditor(true)}
+          onClick={() => {
+            setQuestionEditorDirty(false);
```

Comment:
i don't love that we set this in every file. is there a way to do this within QuestionEditor?

Reply:
I wasn't able to get this to work consistently. I'll give it another shot tho

Reply:
Got this to work!